packages/camera/camera_avfoundation/example/ios/RunnerTests/CameraPermissionTests.m

```diff
@@ -105,21 +138,22 @@ - (void)testRequestCameraPermission_completeWithErrorIfUserDenyAccess {
                           message:@"User denied the camera access request."
                           details:nil];
 
-  id mockDevice = OCMClassMock([AVCaptureDevice class]);
-  OCMStub([mockDevice authorizationStatusForMediaType:AVMediaTypeVideo])
-      .andReturn(AVAuthorizationStatusNotDetermined);
+  self.mockService.authorizationStatusStub = ^AVAuthorizationStatus(AVMediaType mediaType) {
+    XCTAssertEqualObjects(mediaType, AVMediaTypeVideo);
+    return AVAuthorizationStatusNotDetermined;
+  };
 
-  // Mimic user choosing "deny" in permission dialog.
-  OCMStub([mockDevice requestAccessForMediaType:AVMediaTypeVideo
-                              completionHandler:[OCMArg checkWithBlock:^BOOL(void (^block)(BOOL)) {
-                                block(NO);
-                                return YES;
-                              }]]);
-  FLTRequestCameraPermissionWithCompletionHandler(^(FlutterError *error) {
+  // Mimic user choosing "allow" in permission dialog.
```

Comment:
just double check - is it deny? (`handler(NO)`) below.

Reply:
You're right, thanks for the catch